### PR TITLE
feat: use generic api interface

### PR DESCRIPTION
**Pull request by anbraten**

### CODE REVIEW

File: app/src/components/popups/StopPopup.vue [hunk 1]
```diff
@@ -152,49 +153,37 @@ const augmentedArrivals = computed<(Omit<StopArrival, 'eta'> & { nextStopName?:
...
-      void unsubscribe(subject);
-    }
-    subject = `data.map.stop.${newMarker.id}`;
-    await subscribe(subject, stops);
-  },
-  { immediate: true },
-);
-
-const tripSubscriptions = new Set<string>();
+// const tripSubscriptions = new Set<string>();
```
Comment (lukashass) [inline app/src/components/popups/StopPopup.vue:186]: @anbraten Is this no longer needed?

File: app/src/components/popups/StopPopup.vue [hunk 2]
```diff
@@ -152,49 +153,37 @@ const augmentedArrivals = computed<(Omit<StopArrival, 'eta'> & { nextStopName?:
...
+        .map((a) => {
+          // const trip = trips.value[a.tripId];
+
+          // let nextStopName: string | undefined;
+          // if (trip !== undefined && trip.arrivals !== undefined) {
+          //   const nextStopIndex = trip.arrivals.findIndex((s) => s.id === props.marker.id);
+          //   if (nextStopIndex !== -1) {
+          //     nextStopName = trip.arrivals[nextStopIndex + 1]?.name;
+          //   }
+          // }
```
Comment (lukashass) [inline app/src/components/popups/StopPopup.vue:174]: @anbraten Does this feature not work with the new api?

**Review by lukashass — APPROVED** [review]

File: app/src/components/popups/StopPopup.vue (hunk 2 above)
Comment (anbraten) [reply app/src/components/popups/StopPopup.vue:174]: @lukashass The nextStop feature was disabled in general, so I commented this code piece for now as well.

File: app/src/components/popups/StopPopup.vue (hunk 1 above)
Comment (anbraten) [reply app/src/components/popups/StopPopup.vue:186]: Also part of the nextStop feature which we disabled.

File: app/src/components/popups/StopPopup.vue (hunk 2 above)
Comment (lukashass) [reply app/src/components/popups/StopPopup.vue]: I think it was still active, just relying on the data already loaded by viewing a bus before viewing a stop. But it's not too important.

**Review by lukashass — APPROVED** [review]

**Pull request merged**